apps/marketplace/pages/SellerAssessmentViewPage.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+import SellerAssessmentView from 'marketplace/components/SellerAssessment/SellerAssessmentView'
+import { loadDomainEvidenceData } from '../actions/supplierActions'
+
+class SellerAssessmentViewPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loading: false
+    }
+  }
+
+  componentDidMount() {
+    if (this.props.match.params.evidenceId) {
+      this.getEvidenceData()
+    }
+  }
+
+  getEvidenceData() {
+    this.setState({
+      loading: true
+    })
+    return this.props.loadInitialData(this.props.match.params.evidenceId).then(response => {
+      this.setState({
+        loading: false
+      })
+      return response.data
+    })
+  }
+
+  render() {
+    if (this.state.loading) {
+      return <LoadingIndicatorFullPage />
+    }
+    return <SellerAssessmentView evidence={this.props.evidence} />
```

Comment:
Do you need an error component in case something goes wrong when loading the data?